contents/tutorindigo.rst

```diff
@@ -0,0 +1,245 @@
+
+- Clone the Tutor Indigo by running the command ``git clone https://github.com/overhangio/tutor-indigo.git``
+- Add your styles in the ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss``
+
+.. code-block:: sass
+
+    body {
+        min-height: initial;
+        background-color: #fff;
+    }
+
+- Install the updated Tutor Indigo Plugin
+
+
+Updating HTML templates
+-----------------------
+
+Add HTML files in ``tutorindigo/templates/indigo/lms/templates``, ensuring folder structure matches `edx-platform/lms/templates <https://github.com/openedx/edx-platform/tree/master/lms/templates>`_ for proper overriding.
+
+For example, You want to add detail of Open edX in footer. You have to follow the below points:
+
+- Clone the Tutor Indigo by running the command ``git clone https://github.com/overhangio/tutor-indigo.git``
+- Search the template in `edx-platform/lms/templates <https://github.com/openedx/edx-platform/tree/master/lms/templates>`_  which is used for footer rendering.
+- The footer template exists at ``edx-platform/lms/templates/footer.html`` in edx-platform.
+- You have to copy the file and paste in Tutor Indigo Plugin at ``tutor-indigo/tutorindigo/templates/indigo/lms/templates/footer.html``
+- Make your changes and install the updated plugin for reflecting the changes.
+
+.. image:: /academy/static/images/tutorindigo/footer-update-1.png
+  :width: 800
+  :alt: Footer Update Image 1
+
+.. image:: /academy/static/images/tutorindigo/footer-update-2.png
+  :width: 800
+  :alt: Footer Update Image 2
+
+Changing MFE Styles
+-------------------
+
+Clone the `@edx/brand <https://github.com/openedx/brand-openedx>`_ package and customize it. Include customized brand package links in ``tutorindigo/plugin.py`` to modify MFE styles. You can checkout `this link <https://github.com/overhangio/tutor-indigo?tab=readme-ov-file#cant-override-styles-using-indigo-theme-for-mfes>`_ for further details.
+
+Activating Dark theme
+---------------------
+
+You can now give users a more familiar experience by enabling the dark theme. Dark themes are increasingly popular for their aesthetic appeal and reduced eye strain, especially in low-light environments. By offering a dark theme, you cater to user preferences and enhance their overall experience on your platform.
+
+To enable the dark theme using Tutor Indigo, set the configuration variable by running the command ``tutor config save --set INDIGO_ENABLE_DARK_THEME=True``. Then follow the "How to deploy Customizations to Production" section at the last of this guide. This simple customization can make a significant difference in user satisfaction and engagement.
+
+How to deploy Customizations to Production
+==========================================
+
+To deploy the above customizations to production, rebuild the "openedx" Docker image and restart containers::
```

Comment:
This whole section is unnecessary. I think that we should move instructions on how to apply changes to a dedicated subsection, as described in my other comment.